js/particles/view/ParticlesScreenView.ts

```diff
@@ -15,17 +15,23 @@ import TimeControlNode from '../../../../scenery-phet/js/TimeControlNode.js';
 import { Node } from '../../../../scenery/js/imports.js';
 import ExampleSimConstants from '../../common/ExampleSimConstants.js';
 import exampleSim from '../../exampleSim.js';
+import ParticlesModel from '../model/ParticlesModel.js';
 import ParticleNode from './ParticleNode.js';
 
 // The model is in nanometers, and this is the number of nanometers per 1 unit in the view.
 const NANOMETERS_PER_PIXEL = 100;
 
-class ParticlesScreenView extends ScreenView {
+export default class ParticlesScreenView extends ScreenView {
+
+  private model: ParticlesModel;
+
+  // the parent for all ParticleNode instances
+  private particlesNode: Node;
 
   /**
-   * @param {ParticlesModel} model - the top-level model for this screen
+   * model - the top-level model for this screen
    */
-  constructor( model ) {
+  public constructor( model: ParticlesModel ) {
```

Comment:
You can actually declare and define the class field here like so: 
```
public constructor( private readonly model: ParticlesModel )
```

Reply:
This would reduce code in the class field declarations as well as the body of the constructor

Reply:
Wow, I didn't know you could do that. How do you decide whether to do so or not? Seems like the fields should all be one way of the other.

Reply:
Okay, done.